packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,409 @@
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const DerivEZDesktopDownload = () => {
+    return (
+        <React.Fragment>
+            <a className='cfd-trade-modal__dxtrade-button' href={''} target='_blank' rel='noopener noreferrer'>
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandDerivEzWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const CTraderDesktopDownload = () => {
+    return (
+        <React.Fragment>
+            <a
+                className='cfd-trade-modal__dxtrade-button'
+                href={'https://ct.deriv.com/'}
+                target='_blank'
+                rel='noopener noreferrer'
+            >
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandCtraderWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const TradeModal = ({
+    mt5_trade_account,
+    is_eu_user,
+    onPasswordManager,
+    toggleModal,
+    dxtrade_tokens,
+    is_demo,
+    platform,
+}: TradeModalProps) => {
+    const differentQRCodes = (acc_type: TPlatformType) => {
+        if (acc_type === 'dxtrade') {
+            return (
+                <>
+                    <QRCode
+                        value={mobileDownloadLink(acc_type, 'android')}
+                        size={5}
+                        style={{ height: 'auto', maxWidth: '100%', width: qr_code_mobile }}
+                    />
+                    <Text align='center' size='xxs'>
+                        {localize('Scan the QR code to download Deriv X.')}
+                    </Text>
+                </>
+            );
+        } else if (acc_type === 'ctrader') {
+            return (
+                <>
+                    <QRCode
+                        value={mobileDownloadLink(acc_type, 'android')}
+                        size={5}
+                        style={{ height: 'auto', maxWidth: '100%', width: qr_code_mobile }}
+                    />
+                    <Text align='center' size='xxs'>
+                        {localize('Scan the QR code to download cTrader.')}
+                    </Text>
+                </>
```

Comment:
This is being duplicated in multiple places so create a more atomic component and separate it out from here.
It will help with code readability.

Reply:
Will do